src/category_theory/sites/sieves.lean

```diff
@@ -138,13 +138,19 @@ begin
     exact bind_comp _ (of_arrows.mk _) (of_arrows.mk _) }
 end
 
+/-- Given a presieve on `F(X)`, we can define a presieve on `X` by taking the preimage via `F`. -/
+def functor_pullback {X : C} (R : presieve (F.obj X)) : presieve X := λ _ f, R (F.map f)
+
+@[simp] lemma functor_pullback_apply {X : C} (R : presieve (F.obj X)) {Y} (f : Y ⟶ X) :
```

Comment:
Maybe call this `mem_functor_pullback`? This would be consistent with the use of `mem` in the rest of this file.

Reply:
But `Inf_apply`, `Sup_apply` etc, and the lemma generated via `simps` all ends in `_apply`?

Reply:
You're right, it's not 100% consistent already. Although the `_apply` in `Inf_apply` could refer to `Inf` being applied, not the presieve. On the other side, we have `id_mem_iff_eq_top`, `pullback_eq_top_of_mem` and `pushforward_le_bind_of_mem`. I would `mem` is better as it highlights that presieves are collections of sets, even though we often write it as function application.

Reply:
Sure, I can rename it If that's the case.